src/components/Settings/SettingsAddressbookShare.vue

```diff
@@ -97,6 +97,9 @@ export default {
 			return t('contacts', 'No users or groups')
 		}
 	},
+	mounted() {
```

Comment:
I like this fix :)
Please just add a small comment stating why this is here as it can be a bit tricky ;)